src/App.jsx

```diff
@@ -41,6 +43,8 @@ const App = () => (
       <Route path="/verifyAccount/:token" element={<AccountVerfication />} />
       <Route path="/verifyAccount" element={<AccountVerfication />} />
       <Route path="/Payments" element={<CheckoutCard />} />
+      <Route path="/Profile/Updatephoto" element={<UpdatePhoto />} />
+      <Route path="/Account/ChangePassword" element={<ChangePassword />} />
```

Comment:
I recommend using lowercase paths, they are more SEO friendly.
```suggestion
      <Route path="/profile/update-photo" element={<UpdatePhoto />} />
      <Route path="/account/change-password" element={<ChangePassword />} />
```